covid_act_now/delphi_covid_act_now/run.py

```diff
@@ -106,4 +107,8 @@ def run_module(params):
             print(f"Failed to archive '{exported_file}'")
 
     elapsed_time_in_seconds = round(time.time() - start_time, 2)
-    logger.info("Completed indicator run", elapsed_time_in_seconds=elapsed_time_in_seconds)
+    logger.info("Completed indicator run",
+                elapsed_time_in_seconds=elapsed_time_in_seconds,
+                csv_export_count=files_exported,
+                earliest_export_date=earliest.strftime("%Y-%m-%d"),
+                latest_export_date=latest.strftime("%Y-%m-%d"))
```

Comment:
`earliest` and `latest` are local mini/maximums, you probably want a global one here

Reply:
:+1: